packages/sdk-v2/src/sdk/mpbridge/constants.ts

```diff
@@ -0,0 +1,217 @@
+import { ethers } from "ethers";
+import { SupportedChains } from "../constants";
+
+/**
+ * Bridge configuration constants
+ */
+export const BRIDGE_CONSTANTS = {
```

Comment:
@romeoscript more hardcoded values with should be read from contract
for these values we just call the contract.
see
https://celoscan.io/address/0x165aEb4184A0cc4eFb96Cb6035341Ba2265bA564#readProxyContract 
`bridgeLimits` & `BRIDGE_TOPIC`